src/ui/components/Search/component.ts

```diff
@@ -37,14 +37,20 @@ export default class Home extends Component {
 
   @tracked('args')
   get isSearchDisabled(): boolean {
-    return this.args.isSSR || !this.args.isOnline;
+    return !this.args.isOnline;
   }
 
   @tracked('locations', 'searchTerm', 'coordinates')
   get showRecent(): boolean {
     return this.locations.length > 0 && !this.searchTerm && !this.coordinates;
   }
 
+  @tracked('coordinates')
+  get notFoundCoordinates(): boolean {
+    let { coordinates } = this;
+    return coordinates === '200,200';
```

Comment:
I think we could use a union type for the coordinates so that we can express we either expect an array of strings or e.g. the number `-1` (which stands for location not found then). I think that would make it easier than using this magic value?

```typescript
coordinates: string[] | number
```